metaflow/datastore/flow_datastore.py

```diff
@@ -68,7 +68,14 @@ def datastore_root(self):
         return self._storage_impl.datastore_root
 
     def get_latest_task_datastores(
```

Comment:
rename as `get_task_datastores`

Reply:
should we also keep `get_latest_task_datastores` which can now just be a call to `get_task_datastores(..., attempt=None, include_prior=False)`

or is `get_latest_task_datastores` not considered public and we should be fine to ditch it? OSS codebase only has two references to this so the internal rename is simple enough. Slightly concerned if something external is relying on this though.

Reply:
renamed based on discussions.